src/draw/DefaultDrawer.js

```diff
@@ -37,7 +53,7 @@ class DefaultDrawer {
     const componentsSelection = this.d3.select(parentId)
       .selectAll('.component')
       .data(components, (data) => data);
-    const svgContainer = componentsSelection.enter().append('svg');
+    const svgContainer = componentsSelection.enter().append('svg').style('border-radius', '1px');
```

Comment:
I think it's not really "pretty" to define the border-radius here without having the context.